user_guide_src/source/changelogs/v4.3.0.rst

```diff
@@ -43,6 +43,7 @@ Others
 - The ``spark`` file has been changed due to a change in the processing of Spark commands.
 - ``InvalidArgumentException`` that is a kind of ``LogicException`` in ``BaseBuilder::_whereIn()`` is not suppressed by the configuration. Previously if ``CI_DEBUG`` was false, the exception was suppressed.
 - ``RouteCollection::resetRoutes()`` resets Auto-Discovery of Routes. Previously once discovered, RouteCollection never discover Routes files again even if ``RouteCollection::resetRoutes()`` is called.
+- ``CITestStreamFilter::$buffer = ''`` no longer causes the filter to be registered to listen for streams. Now there is a ``CITestStreamFilter::registration()`` method for this.
```

Comment:
Ah, this change may break user's existing tests. We need to add *Upgrading Guide*, too.

See also https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#upgrading-guide

Reply:
What if this text is simply moved to the upgrade guide?

Reply:
No. The changelog shows what's changed.
The upgrading guide shows what to do when upgrading.

When you are upgrading and need to change existing code, we must include it in the upgrading guide, even if it is the same thing.

https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#changelog
https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#upgrading-guide

Reply:
I hope I succeeded

Reply:
Thanks. Perfect!